server/CGameHandler.cpp

```diff
@@ -5414,26 +5414,29 @@ void CGameHandler::runBattle()
 				continue;
 			}
 
-			if(next->getCreature()->idNumber == CreatureID::CATAPULT && (!curOwner || curOwner->getSecSkillLevel(SecondarySkill::BALLISTICS) == 0)) //catapult, hero has no ballistics
+			if( next->getCreature()->idNumber == CreatureID::CATAPULT )
```

Comment:
No spaces after "(" and before ")", please. http://wiki.vcmi.eu/index.php?title=Coding_guidelines